controllers/rollout/trafficrouting/trafficroutingutil.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2022 The Kruise Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package trafficrouting
+
+// TrafficRoutingController common function across all TrafficRouting implementation
+type TrafficRoutingController interface {
+	SetRoutes(desiredWeight int32) error
+
+	VerifyTrafficRouting(desiredWeight int32) (bool, error)
+
+	DoFinalising() error
```

Comment:
VerifyTrafficRouting -> Verify
DoFinalising -> finalize

and plz comment about the params and return variables